styles/style.css

```diff
@@ -81,20 +89,146 @@ textarea {
     appearance: none;
     border-radius: 0;
 }
+
 .btn {
     display: flex;
     justify-content: center;
-	align-items: center;
+    align-items: center;
     padding: 8px;
     width: 100%;
     max-width: 94px;
     border-radius: 2px;
     color: var(--color-white);
     font: var(--font-button-default);
 }
+
 .btn.btn--primary {
     padding: 16px;
     max-width: 163px;
     background: var(--color-accent);
     font: var(--font-button-small);
 }
+
+ul {
+    list-style-type: none;
+}
+
+a {
+    text-decoration: none;
+}
+
+.container {
+    margin: 0 auto;
+    width: 100%;
+    max-width: 1177px;
+}
+
+.site-header .header-top {
+    padding: 14px 0;
+    background-color: #7E33E0;
+}
+.site-header .header-top a {
```

Comment:
make hover effect like in header-bottom a links

Reply:
corrected

Reply:
Icons at the top should always change their color to red (as at Home link). So by default those icons have a white color. After hovering on them they should be red as the links